firecracker/kernel/build.sh

```diff
@@ -35,18 +35,25 @@ readonly BUILD_DIR
 # Copy kernel into dist.
 ########################################
 readonly KERNEL_BIN_DIR="${BUILD_DIR}/firecracker/build/kernel/linux-${KERNEL_VERSION}/"
-readonly KERNEL_BIN_FILE="vmlinux-${KERNEL_VERSION}-x86_64.bin"
+readonly KERNEL_BIN_FILE="${KERNEL_BIN_DIR}/vmlinux-${KERNEL_VERSION}-x86_64.bin"
```

Comment:
I take it this name is determined by Firecracker's `devtool` script, and not something we control, yes?

Reply:
Correct